internal/pkg/client/ocisif/ocisif.go

```diff
@@ -16,7 +16,7 @@ import (
 
 	ocitypes "github.com/containers/image/v5/types"
 	"github.com/google/go-containerregistry/pkg/name"
-	v1 "github.com/google/go-containerregistry/pkg/v1"
+	ggcr "github.com/google/go-containerregistry/pkg/v1"
```

Comment:
I'd prefer `ggcrv1` if you are going to change this - we have other packages from ggcr in here too, and the `v1` bit is important to know.

Reply:
Will do. The code that says `v1.Xyz` just looks very awkward to me - as in, it prompts the question "v1 of _what_?"